lunes_cms/cms/models/static.py

```diff
@@ -66,6 +66,21 @@ def convert_umlaute_audio(instance, filename):
     return create_ressource_path("audio", filename)
 
 
+def upload_sponsor_icons(instance, filename):
+    """
+    Upload path for sponsor icons
+
+    :param instance: instance where the current file is being attached
+    :type instance: django.db.models
+    :param filename: name of the file
+    :type filename: str
+
+    :return: file path for sponsor icons
+    :rtype: str
+    """
+    return create_ressource_path("sponsors", filename)
```

Comment:
In a chat with @juliawohlhueter we thought it was better not to return the pathname but instead return a boolean and according icon (s. screenshot), if there is a logo or not. The same mechanism we have for vocabularies (incl. icon). 
![grafik](https://user-images.githubusercontent.com/72705147/219876771-ed356fdb-fff9-47df-bc87-0b28476d8067.png)